app.rb

```diff
@@ -0,0 +1,31 @@
+require 'sinatra/base'
+require_relative './models/peep'
+
+class ChitterApp < Sinatra::Base 
+
+  enable :sessions
+
+  get '/' do
+    @peeps = Peep.list_ordered_peeps
+    erb :index
+  end
+
+  post '/sign-in' do
+    session[:logged_in] = true
+    session[:username] = params[:username]
+    redirect to '/'
+  end
+
+  post '/create-peep' do
+    Peep.create(message: params[:message], creator: session[:username])
+    redirect to '/'
+  end
+
```

Comment:
Skinny controller with clear method and param names being used :) easy to follow the code!